### DISCUSSION TITLE
Order of presenting individual work

### DISCUSSION THREAD

**Discussion by neelima-girlcoder** [aryashah2k/Programming-Patterns-And-Designs#69] [Q&A] [answered]
Do we attach test screenshots for each pattern that we write code for?

**Comment by aryashah2k** [comment, marked as answer]
Hi, so basically what needs to be done is you can test the working of your code snippet locally and attach screenshots of the patterns if you wish to. Since this will be too cumbersome, I am allowing not to attach working screenshots of each. In turn you can test your work locally and only then open a PR stating that it has been tested locally so that while reviewing, things will be easy for me.
Have great day!
